tensorflow/core/kernels/where_op.cc

```diff
@@ -66,7 +66,7 @@ int64 CountAccumulator(const T* begin, const T* end) {
 
 template <>
 int64 CountAccumulator<bool>(const bool* begin, const bool* end) {
-  return std::accumulate(begin, end, 0LL);
+  return std::count_if(begin, end, [](bool value) { return value; });
```

Comment:
Does `std::count(begin, end, true)` not work?  (I would expect the casting to bool to happen before the values are compared, and I believe `static_cast<bool>(1) == static_cast<bool>(2)` should be true?)

If using `std::count` does work, then I'd prefer that, since it's marginally more readable. (I expect that they compile down to the same thing.)

If not, then this is fine as-is.

Reply:
std::count had same issue, unfortunately.

Reply:
Ok thanks for checking. This looks good to me then.